rules/linux/persistence_systemd_scheduled_timer_created.toml

```diff
@@ -176,6 +174,10 @@ file where host.os.type == "linux" and event.action in ("rename", "creation") an
     "/opt/puppetlabs/puppet/bin/ruby", "/usr/libexec/platform-python", "/kaniko/kaniko-executor",
     "/usr/local/bin/dockerd", "/usr/bin/podman", "/bin/install", "/proc/self/exe"
   ) or
+  process.name like (
+    "python*", "crio", "apt-get", "install", "snapd", "cloudflared", "sshd", "convert-usrmerge", "docker-init",
+    "google_metadata_script_runner"
```

Comment:
google_metadata_script_runner is the equivalent of AWS's user-data scripts. Typically bash scripts meant to run at startup for a cloud instance. Startup scripts are a great persistence mechanism in cloud instances so while we ignore here, we could potentially look at a rule to identify anomalous cloud instance startup scripts later on 🤷🏽

Reply:
I could remove those from the tuning. Those weren't that big in terms of FPs. If this tuning introduces cloud evasion opportunities I will get it out of this and other rules!